lib/src/main/java/org/team100/lib/config/NotePoseDetector.java

```diff
@@ -69,9 +65,9 @@ public Rotation2d fieldRelativeAngleToNote() {
         switch (Identity.instance) {
             case BETA_BOT:
             case COMP_BOT:
+            case BLANK:
```

Comment:
it looks to me like there's an infinite loop here: fieldRelativeAngleToNote calls fieldRelativePose2d which callsFieldRelativeAngleToNote.

Reply:
I guess.  the main issue is just way too much code here, you can remove most of these methods.